README.md

```diff
@@ -76,6 +62,32 @@ Create a profile, review the sample use cases to familiarize yourself with the c
 
 You can now use all the functionalities of the extension.
 
+### Use Base Profiles
+
+Leverage existing base profiles with a token to log in to API Mediation Layer.
+
+Before using the base profiles functionality, ensure that you have [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) installed.
+
```

Comment:
Version? Should be the Zowe CLI part of Zowe v1.13.0 or higher

Reply:
I refer them to the appropriate doc. They always have the latest version

Reply:
I'll just add the version. nvm